chia/cmds/init_funcs.py

```diff
@@ -107,6 +108,38 @@ def check_keys(new_root: Path) -> None:
     save_config(new_root, "config.yaml", config)
 
 
+#
+# Checks if the given key is used for either the farmer rewards or pool rewards
+# returns a tuple of two booleans
+# The first is true if the key is used as the Farmer rewards, otherwise false
+# The second is true if the key is used as the Pool rewards, otherwise false
+# Returns both false if the key cannot be found with the given fingerprint
+#
+def check_key_used_for_rewards(new_root: Path, sk: PrivateKey, max_ph_to_search: int) -> Tuple[bool, bool]:
```

Comment:
This method doesn't belong here. cmds is for command line program code only. And init_funcs is for things called under the command `chia init`

Reply:
I added it here because of the rather similar check_keys and I thought it might be also used for cmd line "delete key" functionality in the future. Would a private function inside wallet_rpc_api be suitable? Otherwise, can you suggest something?

Reply:
Code updated, I added it all inside wallet_rpc_api